libs/external-router/guides/configuration.md

```diff
@@ -0,0 +1,25 @@
+# `@daffodil/external-router` Configuration
+
+The External Router exposes a configuration token that you can provide in order to configure your the behavior of the package.
```

Comment:
Should this also be configurable via an arg to a module?

Reply:
Yes, it already was, good catch. 2a9fc4808544b6d3b5512c6b748cda955ce2c525